src/mscorlib/model.xml

```diff
@@ -7389,7 +7389,7 @@
     </Type>
     <Type Name="System.Threading.ThreadStartException">
       <Member Status="ImplRoot" Name="#ctor(System.Exception)" /><!-- EE -->
-      <Member Name="#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)" />
+      <Member Status="ImplRoot" Name="#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)" />
```

Comment:
My understanding (though I could be wrong) is that it's used for non-public APIs that must not be tree-shaken away.  @weshaggard?